www/src/components/headSeo.astro

```diff
@@ -15,7 +15,16 @@ const formattedContentTitle = frontmatter
   : SITE.title;
 
 const imageSrc = frontmatter?.image?.src ?? OPEN_GRAPH.image.src;
-const imageUrl = new URL(imageSrc, Astro.url);
+
+let imageUrl;
+
+// Check if image is a relative path or absolute URL, decide how to handle it accordingly
+if (imageSrc.startsWith("/")) {
```

Comment:
I just merged the other PR for this, can you merge next to your branch and make sure it's all good?